src/script/auth/module/action/AuthAction.ts

```diff
@@ -356,9 +346,6 @@ export class AuthAction {
           throw new Error(`Could not find value for '${LocalStorageKey.AUTH.PERSIST}'`);
         }
         const clientType = persist ? ClientType.PERMANENT : ClientType.TEMPORARY;
-        if (isTemporaryClientAndNonPersistent(clientType === ClientType.PERMANENT)) {
```

Comment:
This logic has moved there https://github.com/wireapp/wire-webapp/pull/12968/files#diff-48632c70319182040bb02263c6e38bce17b438bcb0f2ba730020ad83f14ab7a5R32